oca_dependencies.txt

```diff
@@ -3,3 +3,4 @@
 product-attribute
 server-ux
 web
+stock_inventory_discrepancy https://github.com/ursais/stock-logistics-warehouse bodedra_12.0-mig-stock_inventory_discrepancy
```

Comment:
@osimallen You should remove this and wait for the module to be merged and rebase this branch.

Reply:
@max3903 I added those line. Yes will do.